.github/workflows/ci.yml

```diff
@@ -163,12 +165,38 @@ jobs:
                      -DCMAKE_INSTALL_PREFIX=${GITHUB_WORKSPACE}/install/deps ..
         cmake --build . --config ${{ matrix.build_type }} --target install
 
+        # osqp
+        cd ${GITHUB_WORKSPACE}
+        git clone --recursive -b ${osqp_TAG} https://github.com/oxfordcontrol/osqp
+        cd osqp
+        mkdir -p build
+        cd build
+        cmake -A x64 -DCMAKE_TOOLCHAIN_FILE=${VCPKG_INSTALLATION_ROOT}/scripts/buildsystems/vcpkg.cmake \
+                     -DCMAKE_PREFIX_PATH=${GITHUB_WORKSPACE}/install/deps \
+                     -DCMAKE_BUILD_TYPE=${{ matrix.build_type }} \
+                     -DCMAKE_INSTALL_PREFIX=${GITHUB_WORKSPACE}/install/deps ..
+        cmake --build . --config ${{ matrix.build_type }} --target install
+
+        # osqp-eigen
+        cd ${GITHUB_WORKSPACE}
+        git clone https://github.com/robotology/osqp-eigen
+        cd osqp-eigen
+        git checkout ${OsqpEigen_TAG}
+        mkdir -p build
+        cd build
+        cmake -A x64 -DCMAKE_TOOLCHAIN_FILE=${VCPKG_INSTALLATION_ROOT}/scripts/buildsystems/vcpkg.cmake \
+                     -DCMAKE_PREFIX_PATH=${GITHUB_WORKSPACE}/install/deps \
+                     -DCMAKE_BUILD_TYPE=${{ matrix.build_type }} \
+                     -DCMAKE_INSTALL_PREFIX=${GITHUB_WORKSPACE}/install/deps ..
+        cmake --build . --config ${{ matrix.build_type }} --target install
+
+
         # CasADi
         # We override the casadi installation structure to be compliant with the folder structure used for
         # the other dependencies. Please check:
         # https://github.com/casadi/casadi/blob/6f122ca22e2a869903628c5738f154c8ac0f7455/CMakeLists.txt#L317
         cd ${GITHUB_WORKSPACE}
-        git clone https://github.com/casadi/casadi.git -b ${CasADi_TAG} casadi
+        git clone https://github.com/GiulioRomualdi/casadi.git -b ${CasADi_TAG} casadi
```

Comment:
Maybe we should move that to ``dic`` to reduce the bus factor.

Reply:
c6a135a13f17c52155951cf0a032b01d3ed545c9